docs/sources/alerting/set-up/provision-alerting-resources/edit-provisioned-resources/_index.md

```diff
@@ -0,0 +1,47 @@
+---
+canonical: https://grafana.com/docs/grafana/latest/alerting/set-up/provision-alerting-resources/edit-provisioned-resources/
+description: Export and edit provisioned resources in Grafana
+keywords:
+  - grafana
+  - alerting
+  - alerting resources
+  - provisioning
+  - export
+  - edit
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
+menuTitle: Export and edit provisioned resources in Grafana
+title: Export and edit provisioned alerting resources in Grafana
+weight: 400
+---
+
+# Export and edit provisioned alerting resources in Grafana
```

Comment:
How about having two separate sections one for `Export` and the second one for `Modify Export`?

We give users the ability to export on three different levels. The user can export:
- All Grafana-managed alert rules
- Single folder
- Single group

It would be nice to show document that

`Modified export` could be a separate section because this is relevant only to an alert rule